package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "bandapixels-test-blog",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./src/config/server/index.ts",
+  "scripts": {
+    "build": "tsc --skipLibCheck",
+    "start": "ts-node ./src/config/server/index.ts"
```

Comment:
Here should be script named somehow like `start:watch` which will trigger `nodemon`